Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,53 @@
-/*  Start coding below for POSITION THE CUBES TASK*/
+/*  Start coding below for POSITION THE CUBES TASK*/
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.parent{
+    position: relative;
+    justify-content: center;
+    height: 400px;
+    width: 400px;
+    background-color: #675BC6;
+}
+
+.child-1{
+    height: 20%;
+    width: 20%;
+    border: 2px solid white;
+    top:0px;
+    background-color: #60B177;
+}
+
+.child-2{
+    height: 20%;
+    width: 20%;
+    top: 0px;
+    right: 0px;
+    border: 2px solid white;
+    background-color: #ED6D52;
+}
+
+.child-3{
+    height: 10%;
+    width: 10%;

```

Comment:
The 'place-items' property is not applicable for positioning elements. Consider using 'position' property instead.